ddl/column.go

```diff
@@ -417,3 +417,40 @@ func (d *ddl) backfillColumn(t table.Table, columnInfo *model.ColumnInfo, handle
 
 	return nil
 }
+
+func (d *ddl) onModifyColumn(t *meta.Meta, job *model.Job) error {
+	tblInfo, err := d.getTableInfo(t, job)
+	if err != nil {
+		return errors.Trace(err)
+	}
+	newCol := &model.ColumnInfo{}
+	err = job.DecodeArgs(newCol)
+	if err != nil {
+		job.State = model.JobCancelled
+		return errors.Trace(err)
+	}
+	var done bool
+	for i, col := range tblInfo.Columns {
```

Comment:
`findCol` returns a *model.ConlumnInfo, but we need to replace that column with new column.